omod/src/main/java/org/openmrs/module/patientflags/extension/html/FlagPatientOverviewExt.java

```diff
@@ -32,8 +32,7 @@ public String getRequiredPrivilege() {
 	
 	@Override
 	public String getContent() {
-		// TODO Auto-generated method stub
-		return null;
+		return "<br>";

```

Comment:
see the image below when content returned null or empty string. Changing it to any text or html works perfect. The openmrs version is 1.9.7. The test was done both ways i.e downloading module from module repo and running a flag and also by building module with openmrs version 1.9.7. Both shows same output.

![pferror](https://cloud.githubusercontent.com/assets/4829880/6425070/a40adf98-bf38-11e4-84dc-e5394b9091b1.png)